docs/docs/system-administration/system-limits.md

```diff
@@ -38,3 +30,29 @@ Please contact us if you want to override the current system limits.
 
 You can modify the system limits on a per-Project basis. These limits are defined in the database against the Project.
 The easiest way to modify them is with the [Django admin](/deployment/configuration/django-admin.md) interface.
+
+## Traffic Limits
+
+### SDK Limits
+
+Requests made by our SDKs are _not_ rate limited, by design; we can't predict what sort of profile your traffic will
+look like.
+
+#### Free Plan Limits
+
+If you are on our Free Plan, we will stop answering requests from our API if you go over your
+[Free Plan limit](https://www.flagsmith.com/pricing). We will send you a warning email 7 days before we block your
```

Comment:
When is the block automatically cleared? Is it when the next billing period starts (or when the account is upgraded) or something else?

Also, who receives the email? Is it all org admins? There's also in-app notifications about this right?

Reply:
@matthewelwell to answer these? I'm not sure!

Reply:
> When is the block automatically cleared?

30 days after the warning email was first sent, or when they upgrade. 

> Also, who receives the email?

Org admins receive the email if the usage is <100%, all users in the organisation receive it if usage is >= 100%

> There's also in-app notifications about this right?

Yes, there is a warning at the top of the dashboard showing the API usage.